_buildhelper_cython.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python
+
+###################
+# build helper for pyproject.toml to do the
+# cython parts of the build.
+#
+# This was ported from logic originally in the
+# old-stype setup.py, in order to comply with the
+# new PEP-517/PEP-518 build style.
+#
+# We intentionally do not support editable builds, as
+# they are confusing and usually the wrong thing for
+# projects that have a built component, even one as
+# small as oasis). No more `pip install -e .`
```

Comment:
Is this bit about pip install -e ." accurate?

Reply:
I wrote that before our conversation where you said it's something you like, and after the conversation I planned to remove it after whatever code changes were needed to support it (not knowing that we'd get support without any further overriding of methods).

I'll remove the comment.